README.md

```diff
@@ -26,6 +26,26 @@ crafter-cli create-site -e local -s my-site -n My Site \
 
 `crafter-cli sync-from -e local -s editorial -n origin`
 
+Also, you are able to publish an item to live or staging (staging must be enabled first before using that target):
```

Comment:
You can publish content to the Live or Staging targets (Staging must be enabled to publish to it):